docs/examples/promise/03_listen_to_balance_change/index.js

```diff
@@ -8,20 +8,22 @@ async function main () {
   // Create an await for the API
   const api = await ApiPromise.create();
 
-  // Retrieve the initial balance. Since the call has no callback, it is simply a promise
-  // that resolves to the current on-chain value
+  // Retrieve the initial balance. Since the call has no callback, it is simply a
+  // promise that resolves to the current on-chain value
   let previous = await api.query.balances.freeBalance(Alice);
 
-  console.log(`${Alice} has a ${previous} balance`);
-  console.log(`You may leave this example running and start example 06 or transfer any value to ${Alice}`);
+  console.log(`${Alice} has a previous balance of ${previous || '???'}`);
+  console.log(`You may leave this example running and start example 06` +
```

Comment:
Just keep it single instead of mixing concat with template.

Reply:
Ok, but the reason I used concat is because the if there are more than ~100 characters per line (with browser window width as wide as possible) it clips the contents and enables horizontal scroll bar, so the reader of the embedded code snippet has to scroll to the right to read the rest of it.

Reply:
Fair reason.